steepshot_io/graph/views.py

```diff
@@ -153,285 +139,273 @@ def group_steem_golos(self, list_steem, list_golos):
-            values = self.group_steem_golos(values_steem, values_golos)
+            values_steem, values_golos = self._get_data(name_url='posts_average_author', data_x='day', data_y='count_posts', reverse=False)
+            values = self._group_steem_golos(values_steem, values_golos)
             flag_together = True
-            return render(request, self.template_name, {'values': values, 'flag': flag_together})
+            name_1 = 'posts average steem'
+            name_2 = 'posts average golos'
+            return render(request, self.template_name, {'values': values, 'flag': flag_together, 'name_1': name_1, 'name_2': name_2})
 
 
 class CountVotesWeekly(CountPostWeekly):
-    template_name = 'count_posts_weekly.html'
 
     def get(self, request, *args, **kwargs):
         if 'platform' in request.GET:
             platform = request.GET['platform'].lower()
-            values = self._get_data(platform=platform, url='count_votes_weekly', data_x='day', data_y='count_votes')
-            return render(request, self.template_name, {'values': values})
+            name_1 = 'count votes ' + platform
+            values = self._get_data(platform=platform, name_url='count_votes_weekly', data_x='day', data_y='count_votes', reverse=False)
+            return render(request, self.template_name, {'values': values, 'name_1': name_1})
         else:
-            values_steem, values_golos = self._get_data(url='count_votes_weekly', data_x='day', data_y='count_votes')
-            values = self.group_steem_golos(values_steem, values_golos)
+            values_steem, values_golos = self._get_data(name_url='count_votes_weekly', data_x='day', data_y='count_votes', reverse=False)
+            values = self._group_steem_golos(values_steem, values_golos)
             flag_together = True
-            return render(request, self.template_name, {'values': values, 'flag': flag_together})
-
-
-class AverageVotesWeekly(CountVotesWeekly):
-    def get(self, request, *args, **kwargs):
-        if 'platform' in request.GET:
-            platform = request.GET['platform'].lower()
-            values = self._get_data(platform=platform, url='votes_average_weekly', data_x='day', data_y='votes_count')
-            return render(request, self.template_name, {'values': values})
-        else:
-            values_steem, values_golos = self._get_data(url='votes_average_weekly', data_x='day', data_y='votes_count')
-            values = self.group_steem_golos(values_steem, values_golos)
-            flag_together = True
-            return render(request, self.template_name, {'values': values, 'flag': flag_together})
+            name_1 = 'count votes steem'
+            name_2 = 'count votes golos'
+            return render(request, self.template_name,
+                          {'values': values, 'flag': flag_together, 'name_1': name_1, 'name_2': name_2})
+
+
+# class AverageVotesWeekly(CountVotesWeekly):
```

Comment:
Do we need this code in the future? If no - please delete

Reply:
this code need in future